operator/config/crd/bases/operator.cluster.x-k8s.io_bootstrapproviders.yaml

```diff
@@ -0,0 +1,46 @@
+
+---
```

Comment:
Am I wrong or all the operator files are expect to go under `exp/operator`?
Also this seems a duplicate of exp/operator/config/crd/bases/operator.cluster.x-k8s.io_bootstrapproviders.yaml

Reply:
Same for all the files under `operator/`

Reply:
I removed them all, thanks.

Reply:
I still see the files under `operator/...` instead of `exp/operator/...`

Reply:
oh, sorry. I might've pushed some outdated changes.